build.gradle

```diff
@@ -22,15 +22,19 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'mysql:mysql-connector-java:8.0.18'
+	implementation 'com.auth0:java-jwt:3.4.0'
 	testImplementation('org.springframework.boot:spring-boot-starter-test') {
 		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
 	}
 }
 
-def dbHost = System.getenv("DB_HOST")!= null ? System.getenv("DB_HOST") : "127.0.0.1"
+def dbHost = System.getenv("DB_HOST")!= null ? System.getenv("DB_HOST") : "127.0.0.1:3308"
```

Comment:
please check my review on another PR for this issue

Reply:
Have you commented on the other PR? Will fix this. The reason why I made a change here is when I used docker to spin up mysql, the port was defaulted to 3308 for some reason.